src/main/java/seedu/address/model/person/Patient.java

```diff
@@ -104,9 +104,11 @@ public void addPatientAppointment(Appointment appointment) {
     public String patientAppointmentstoString() {
```

Comment:
Feels similar to drAppointmentsToString().
Consider putting this function in Person, then maybe overriding it in the subclasses.